_layouts/withmap.html

```diff
@@ -11,11 +11,13 @@
         <div class="container mapcontainer flex--vertical">
             {% include sites-world-map.html %}
         </div>  
-      <div class="flex">
-        <div class="container flex--vertical">
-        {{content}}
+        <div class="flex">
+           <div class="container flex--vertical">
+
+        {%- include sites.html -%}
```

Comment:
Having the include in the layout, means that no other page or post can re-use this layout (which beats the idea of layouts). Maybe instead we could leave the `{{ content }}` here, and include in the page?

Reply:
I did it this way because the map content is already included in the same way just above. But, the layout is for map plus content, so you've made a fair point.  

I've changed it to the way you suggested...